spec/workers/detect_invariants_daily_check_spec.rb

```diff
@@ -87,7 +88,7 @@
       described_class.new.perform
 
       expect(Sentry).to have_received(:capture_exception).with(
-        described_class::ApplicationSubmittedWithTheSameCourse.new(
+        described_class::ApplicationSubmittedWithTheSameCourseAndSite.new(
           <<~MSG,
             The following applications have been submitted containing the same course choice multiple times
```

Comment:
Should we amend this message to reflect the change in rules?

Reply:
Yes, good spot!